<launch>

  <node name="image_converter" pkg="face_classification" type="video_emotion_color_demo_topics.py" output="screen">
    <param name="image_topic_output" value="/face_detection/image_raw"/>
    <param name="image_topic_input" value="/naoqi_driver/camera/front/image_raw"/>
    <param name="detection_model_path" value="$(find face_classification)/trained_models/detection_models/haarcascade_frontalface_default.xml"/>
    <param name="emotion_model_path" value="$(find face_classification)/trained_models/emotion_models/fer2013_mini_XCEPTION.110-0.65.hdf5"/>
  </node>

</launch>
